server/src/main/java/io/whitefox/api/deltasharing/server/DeltaSharesApiImpl.java

```diff
@@ -13,14 +16,26 @@
 
 public class DeltaSharesApiImpl implements DeltaApiApi {
 
+  private final String DELTA_TABLE_METADATA_CONTENT_TYPE = "application/x-ndjson; charset=utf-8";
   private final DeltaSharesService deltaSharesService;
+  private final TableLoader<DeltaSharedTable> deltaSharedTableTableLoader;
+
   private static final Function<Throwable, Response> exceptionToResponse =
       t -> Response.status(Response.Status.BAD_GATEWAY)
           .entity(new CommonErrorResponse()
-              .errorCode("BAD GATEWAY")
+              .errorCode(Response.Status.BAD_GATEWAY.getReasonPhrase())
               .message(ExceptionUtil.generateStackTrace(t)))
           .build();
 
+  private <T> CompletionStage<Response> optionalToNotFoundAsync(
```

Comment:
Can you share some code between optionalToNotFound and optionalToNotFoundAsync?

Reply:
Sure